### PR TITLE
Add new volunteer Mayank Sharma

### PR DESCRIPTION
**Pull request by Mayank-KS**

### CODE REVIEW

File: data/users/Mayank-KS.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Mayank Sharma",
+  "bio": "Web Dev",
+  "image": "https://avatars.githubusercontent.com/u/62908116?v=4",
+  "categories": [
+    {
+      "slug": "programming | software-development | web-development",
```
Comment (codewithvoid) [inline data/users/Mayank-KS.json:7]: You need to pick one of the category slug. E.g

```
  "categories": [
  {
    "slug": "programming",
    "topics": [
      "JavaScript"
    ]
  },
  {
    "slug": "web-development",
    "topics": [
      "react"
    ]
  }
],
```
 
Sorry for not being super clear.

File: data/users/Mayank-KS.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Mayank Sharma",
+  "bio": "Web Dev",
+  "image": "https://avatars.githubusercontent.com/u/62908116?v=4",
```
Comment (codewithvoid) [inline data/users/Mayank-KS.json:4]: nit: real pic pfp :P

**Comment by codewithvoid** [comment]
Thanks for volunteering Mayank

File: data/users/Mayank-KS.json (hunk 1 above)
Comment (RakeshSangem) [inline data/users/Mayank-KS.json:7]: Hey @Mayank-KS you need to use only one slug.

**Review by RakeshSangem — CHANGES_REQUESTED** [review]

File: data/users/Mayank-KS.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Mayank Sharma",
   "bio": "Web Dev",
-  "image": "https://twitter.com/KsMayank/photo",
+  "image": "https://media-exp1.licdn.com/dms/image/C4D03AQHVR9bNwkTT8Q/profile-displayphoto-shrink_800_800/0/1647273419200?e=2147483647&v=beta&t=9b7AQ2TrUIFchVnNKgVOCVtfAHFE5ofedgq1YDsuP0k",
```
Comment (RakeshSangem) [inline data/users/Mayank-KS.json:4]: Link doesn't working. 

Instead use this link `https://pbs.twimg.com/profile_images/1592462470387728384/QanHwY5i_400x400.jpg`

Comment (Mayank-KS) [reply data/users/Mayank-KS.json:4]: I already solved it with linkedIn pic

Comment (RakeshSangem) [reply data/users/Mayank-KS.json:4]: > I already solved it with linkedIn pic

Yeah, but it is not working, you can in the deployment preview.

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**